features/FxA-107-security-event-history/README.md

```diff
@@ -70,5 +228,63 @@ we will log the `Account.history.seen` message.
   the the tides.
 
 - **Users may wish to revoke an address**: If a user has signed in while
-  in a hotel some where, they may wish to be able to to signal that this
+  in a hotel somewhere, they may wish to be able to signal that this
   IP is really **not** something to be trusted.
+
+
+
+## Results
+
+(This section to be completed once deployed to production)
+
+### Phase 1
+
+When Phase 1 is complete,
+we will include here
+a snapshot of the hit-rate graph.
+
+We will also calculate,
+but not share publicly,
+the absolute number of hits per day
+at each level of recency.
+
+If these numbers seem high enough
+to justify further development,
+we will proceed to Phase 2.
+
+### Phase 2
+
+When Phase 2 is complete,
+we will include here
+a snapshot of the false-positive-rate graph.
+
+We will also calculate,
+but not share publicly,
+the absolute number of false positives per day
+at each level of recency.
+
+If the false-positive rate seems low enough
+then we will proceed to Phase 3.
+
+### Phase 3
+
+When Phase 3 is complete,
+we will include here a revised summary
+of the hit-rate and false-positive-rate.
+
+We will show a snapshot of
+the sign-in success rate graph
+before and after the deployment,
+and comment on any increase
+that it appears to have caused.
```

Comment:
In particular, imagining the graphs that we'd want to produce at the end of this, makes it really obvious that we don't have the ability to do a proper A/B test of it.  We can look at the graph over time and say "look, the sign-in success rate increased at the same time as we rolled out this feature", but we can't say "the sign-in success rate increased for users who had this feature, compared to those who didn't".

If we want an A/B version of this graph, we have additional engineering work to do to enable it to be built.  @davismtl thoughts?